services/workflows-service/src/common/utils/winston-logger/winston-logger.ts

```
@@ -64,7 +64,22 @@ export class WinstonLogger implements IAppLogger {
     if (typeof error === 'string') {
       this.logger.error({ message: error, ...payload });
     } else {
-      this.logger.error({ error, ...payload });
+      const errorProperties = Object.getOwnPropertyNames(error).reduce(
+        (acc, key) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)